ui_framework/components/gallery/gallery_button/gallery_button.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render } from 'enzyme';
+import { requiredProps } from '../../../test/required_props';
+
+import {
+  KuiGalleryButton,
+} from './gallery_button';
+
+test('renders KuiGalleryButton', () => {
+  const component = <KuiGalleryButton { ...requiredProps }>children</KuiGalleryButton>;
```

Comment:
Can we also add an `href` prop here, since this button ends up being an `<a>` element?